### PR TITLE
Gui

### PR DESCRIPTION
**Pull request by satang500**
combine all three tickets: T67443, T64943, T64935
1.  allow only one fs-dead
2. change alert message for fs-dead (also changed message for Queue to Fail )
3. add job links in alias tab

### CODE REVIEW

File: hydra-web/src/js/modules/alias.js [hunk 1]
```diff
@@ -131,7 +131,17 @@ function(
...
+                        if(length == 1) {
+                            return "<a href='#jobs/"+encodeURIComponent(val)+"/conf'>"+val+"</a>";
+                        } else if(length > 1) {
+                            var list = "";
+                            for(i in val)
+                                list += "<a href='#jobs/"+encodeURIComponent(val[i])+"/conf'>"+val[i]+"</a>&nbsp;&nbsp;";
+                            return list;
+                        } else {
+                            return "";
+                        }
```
Comment (tedpearson) [inline hydra-web/src/js/modules/alias.js]: There is no need to handle an array of 1 element separately than more than one element, this only complicates the code.

File: hydra-web/src/js/modules/datatable.js [hunk 2]
```diff
@@ -925,13 +925,15 @@ function(_, $, dt, Backbone){
                 checkbox.attr("class","checkbox checked-all");
                 checkbox.parent().data("state","all");
                 this.views.selectable.find("button.btn-hide-zero").removeClass("disabled");
-            }
```
Comment (tedpearson) [inline hydra-web/src/js/modules/datatable.js]: I find this whole section extremely difficult to understand.

File: hydra-web/src/js/modules/host.js [hunk 3]
```diff
@@ -224,8 +224,15 @@ function(
...
-				msg += "After failing, cluster will go from " + util.formatPercent(data.prefail) + "% disk used to " + util.formatPercent(data.postfail) + "%.\n";	
+
+                var msg = "";
+                if(deadFs) {        // dead
+                    msg = "<span class=\"bold-red\">Do you really want to fail now (file system dead) " + uuids + "?</span><br/>";
+                } else {            // queue
+                    msg = "<span>Do you really want to queue " + uuids + " to fail?</span><br/>";
+                }
+                msg += "(After failing, cluster will go from " + util.formatPercent(data.prefail) + "% disk used to " + util.formatPercent(data.postfail) + "%)";
+
```
Comment (tedpearson) [inline hydra-web/src/js/modules/host.js]: This looks fixed now.

File: hydra-web/src/js/modules/alias.js (hunk 1 above)
Comment (tedpearson) [inline hydra-web/src/js/modules/alias.js]: Looks better now

File: hydra-web/src/js/modules/datatable.js (hunk 2 above)
Comment (tedpearson) [inline hydra-web/src/js/modules/datatable.js]: This is better

**Comment by yuesong** [comment]
@tedpearson, OK to merge? @satang500, Please close the other PRs if this one has all the latest and greatest.

**Pull request merged**